modules/mygaruIdSystem.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * This module adds MyGaru Real Time User Sync to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/mygaruIdSystem
+ * @requires module:modules/userId
+ */
+
+import { ajax } from '../src/ajax.js';
+import { submodule } from '../src/hook.js';
+
+const bidderCode = 'mygaru';
+const syncUrl = 'https://ident.mygaru.com/v2/id';
+
+export function buildUrl(opts) {
+  const queryPairs = [];
+  for (let key in opts) {
+    if (opts[key] !== undefined) {
+      queryPairs.push(`${key}=${encodeURIComponent(opts[key])}`);
+    }
+  }
+  return `${syncUrl}?${queryPairs.join('&')}`;
+}
+
+function requestRemoteIdAsync(url) {
+  return new Promise((resolve) => {
+    ajax(
+      url,
+      {
+        success: response => {
+          const jsonResponse = JSON.parse(response);
+          const { iuid } = jsonResponse;
+          resolve(iuid);
+        },
+        error: () => {
+          resolve();
+        },
+      },
+      undefined,
+      {
+        method: 'GET',
+        contentType: 'application/json'
+      }
+    );
+  });
+}
+
+/** @type {Submodule} */
+export const mygaruIdSubmodule = {
```

Comment:
please add updates to the following files, match existing id module patterns:
modules/.submodules.json
modules/userId/eids.js
modules/userId/eids.md
modules/userId/userId.md

Reply:
modules/userId/eids.js  doesn't contain module specific values anymore